app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="app_name" translatable="false">RoadSigns demo</string>
+    <string name="sign_none">Podepsat Žádný</string>
+    <string name="environmental_badges_content_description_green">Popis obsahu ekologických odznaků zelený</string>
+    <string name="environmental_badges_content_description_yellow_green">Popis obsahu ekologických odznaků žlutozelený</string>
+    <string name="environmental_badges_content_description_red_yellow_green">Environmentální odznaky obsah popis červená žlutá zelená</string>
+    <string name="app_name">Automobily se zákazem nafty jsou zdarma pro obyvatele, dokud nebude otevřeno 5 EUR</string>
+    <string name="app_name">Zákaz dieselovým motorovým vozidlům zdarma, dokud nebude euro v otevřené pro obyvatele Hamburku</string>
+    <string name="app_name">Zákaz nafty zdarma od 5 EUR kromě dodávkových vozidel Stuttgart</string>
+    <string name="app_name">Zákaz auta zdarma do 5 euro benzín do 2 euro darmstadt</string>
+    <string name="app_name">Zákaz osobních automobilů zdarma do 5 EUR benzinu do 2 EUR Darmstadt</string>
```

Comment:
This looks like a copy/paste mistake - the `name` is always `app_name` here but should be as [here](https://github.com/Umweltzone/roadsigns/blob/92a684018c096f1d2d5f2bb4e6a314267548bf36/roadsigns/src/main/res/values/strings.xml).